src/app/icd/client/ICDClientInfo.h

```diff
@@ -0,0 +1,62 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <crypto/CHIPCryptoPAL.h>
+#include <lib/core/CHIPConfig.h>
+#include <lib/core/DataModelTypes.h>
+#include <lib/core/ScopedNodeId.h>
+#include <lib/support/CodeUtils.h>
+#include <stddef.h>
+
+namespace chip {
+namespace app {
+
+struct ICDClientInfo
+{
+    ScopedNodeId peer_node;
+    uint32_t start_icd_counter         = 0;
+    uint32_t offset                    = 0;
+    uint64_t monitored_subject         = static_cast<uint64_t>(0);
+    Crypto::Aes128KeyHandle mSharedKey = Crypto::Aes128KeyHandle();
```

Comment:
Why the weirdly inconsistent naming?  Please use some single naming convention....

Reply:
updated, thanks

Reply:
@yunhanw-google It's still inconsistent: snake_case vs camelCase.  Please pick one thing.

Reply:
I forgot to push this change in..sure, would update it.